### PR TITLE
OCPRHV-773 remove RH-Virtualization from common_attributes file

### PR DESCRIPTION
**Pull request by emarcusRH**
remove RH-Virtualization attributes from common_attributes file

Version(s):    enterprise 4.14

Issue:  https://issues.redhat.com/browse/OCPRHV-773

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
LGTM! However I'm new to reviewing, so please wait until @adellape approves my review before proceeding. Thanks!

**Review by adellape — COMMENTED** [review]
@skopacz1 Peer review LGTM 👍

**Pull request merged**
